Machine_Learning/Dimensionality_Reduction_&_Neural_Networks/ML_6.6_Feedforward_Neural_Networks_(D).md

```diff
@@ -0,0 +1,27 @@
+Title : Feed Forward Neural Network
+This conversation was marked as resolved by ASLManasa
+
+🔴 **Type of Content** : Documentation
+
+🔴 **Domain** : Machine Learning
+
+🔴 **Module** : Dimensionality Reduction & Neural Networks
+
+---
+
+<h1>Neural Networks</h1>
+A Neural network is also known as Artifical Neural Network is a series of algorithms that endeavors to recognize underlying relationships in a set of data through a process that mimics the way bological neurons signal to each other. It is the  In this sense, neural networks refer to systems of neurons, either organic or artificial in nature. These are the heart of the deep learning.
+<br>
+<h1> Feed Forward Neural Network</h1>
+Feed Forward Neural Network is a classification algorithim.It is an artifical neural network in which the nodes doesnt form any cycle or directed acyyclic graph..It is the simplest form of neural network. It opposite is reccurent neural network in which the nodes form a cycle .<br>

```

Comment:
acyyclic -> acyclic 
It -> It's before opposite
and remove one full stop after graph.